eq-author-api/migrations/updateCreatedByToUseUsers.js

```diff
@@ -0,0 +1,25 @@
+//This is an auto-generated file.  Do NOT modify the method signature.
+
+const { getUserByName, getUserByEmail } = require("../utils/datastore");
```

Comment:
As discussed these functions could be moved here as they aren't used in the rest of the app.